io/src/test/resources/org/commonjava/maven/ext/manip/io/pme.yaml

```diff
@@ -0,0 +1,11 @@
+da:
+  configKey: value
+  configHierarchy:
+    configKey1: value1
+
+pme:
+  version.suffix: redhat-1
+  dependencyExclusion.com.google.guava:guava@*: 18.0
+  repo-reporting-removal: true
+  jsonUpdate: >
+    distro/data/src/main/resources/data/basic-settings.apiman.json:$$..apimanVersion:${apiman:version}\,manager/ui/war/npm-shrinkwrap.json:$$..resolved:
```

Comment:
are we going to get into trouble with escape characters, here or in the properties file? If we have an escape character we normally use on the CLI, and presumably YAML has one (I know properties files use '\' for escaping), how will those interact? Will we have things cropping up like: '\\\\\\something'?